pkgs/by-name/de/deltatouch/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, cmake
+, intltool
+, libdeltachat
+, lomiri
+, qt5
+, quirc
+}:
+
+stdenv.mkDerivation {
+  pname = "deltatouch";
+  version = "1.3.1-unstable-2024-02-25";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "lk108";
+    repo = "deltatouch";
+    rev = "dd86a77291a6e9f1c363e7cdf5999e58fb5b84e5";
+    hash = "sha256-03J2YLXF1uDnTmDsrxWjlfDsN7mcc68MbWy/HTIcM2E=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    intltool
+    cmake
+  ];
+
+  patchPhase = ''
+    mkdir -p $out/share
+    mkdir -p $out/lib
+
+    substituteInPlace CMakeLists.txt --replace "set(DATA_DIR /)" "set(DATA_DIR $out/share)"
+    substituteInPlace plugins/DeltaHandler/CMakeLists.txt --replace '/lib/' '${placeholder "out"}/${qt5.qtbase.qtQmlPrefix}'
+    substituteInPlace plugins/DTWebEngineProfile/CMakeLists.txt --replace '/lib/' '${placeholder "out"}/${qt5.qtbase.qtQmlPrefix}'
+    mkdir -p build/plugins/DeltaHandler
+    cp --no-preserve=mode ${libdeltachat}/lib/libdeltachat.so build/plugins/DeltaHandler/libdeltachat.so
+    cp --no-preserve=mode ${quirc}/lib/libquirc.so.1.0 build/plugins/DeltaHandler/libquirc.so.1.2
+  '';
+
+  postInstall = ''
+    mkdir -p $out/bin
+    mv $out/deltatouch $out/bin
+    cp --no-preserve=mode ${libdeltachat}/lib/libdeltachat.so $out/lib/libdeltachat.so
+    cp --no-preserve=mode ${quirc}/lib/libquirc.so.1.0 $out/lib/libquirc.so.1.2
+  '';
+
+  qtWrapperArgs = [
+    ''--set CLICKABLE_DESKTOP_MODE y'' # Hack to make sending/receiving messages work, we don't use clickable
```

Comment:
You're correct on all accounts. It was needed at some point in the past, and it may soon be needed again as I will try to merge the nixpkg branch into main again.

Reply:
Turns out that even after merging it into main, it's not needed anymore to set CLICKABLE_DESKTOP_MODE, so this part can be removed